From b73cba6541f79d536860e71349f80836f1598e55 Mon Sep 17 00:00:00 2001
From: JPratama7 <josepratama080@gmail.com>
Date: Sat, 31 Aug 2024 16:13:14 +0700
Subject: [PATCH] Revert "XANMOD: kconfig: add 500Hz timer interrupt kernel
 config option"

This reverts commit 3af997da2505340391890ae2145752cb5ec18e23.
---
 kernel/Kconfig.hz | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/kernel/Kconfig.hz b/kernel/Kconfig.hz
index 0ff51d4d265c..38ef6d06888e 100644
--- a/kernel/Kconfig.hz
+++ b/kernel/Kconfig.hz
@@ -40,13 +40,6 @@ choice
 	on SMP and NUMA systems and exactly dividing by both PAL and
 	NTSC frame rates for video and multimedia work.
 
-	config HZ_500
-		bool "500 HZ"
-	help
-	500 Hz is a balanced timer frequency. Provides fast interactivity
-	on desktops with great smoothness without increasing CPU power
-	consumption and sacrificing the battery life on laptops.
-
 	config HZ_1000
 		bool "1000 HZ"
 	help
@@ -60,7 +53,6 @@ config HZ
 	default 100 if HZ_100
 	default 250 if HZ_250
 	default 300 if HZ_300
-	default 500 if HZ_500
 	default 1000 if HZ_1000
 
 config SCHED_HRTICK
-- 
2.46.0